Makefile

```diff
@@ -190,29 +226,43 @@ generate-manifests: $(CONTROLLER_GEN) ## Generate manifests e.g. CRD, RBAC etc.
 		output:crd:dir=./config/crd/bases \
 		output:webhook:dir=./config/webhook \
 		webhook
+	$(CONTROLLER_GEN) \
```

Comment:
Since we are breaking these out, should we also break out the clusterctl generation here into a separate target as well?

Reply:
out of scope for this PR imo